### DISCUSSION TITLE
股市的一些想法

### DISCUSSION THREAD

**Discussion by benjaminlihao** [Vespa314/chan.py#29] [General]
美股最能盈利的还是价值投资。选择几家世界级的公司并长期持有，就行。我同事买特斯拉，英伟达，已经实现财务自由。
不过现在外汇管制，我们这种没有美股开户的，资金也不太好出去，问银行，门槛还挺高。
如果你炒美股，我同事经验就很好。他这十年就买过几只股票。特斯拉，台积电，苹果，英伟达，谷歌。他没有刻意看形态，特斯拉一拿就是五六年。很少操作。赚的盆满钵满。在一个良性市场，选几个世界级的公司，长期来看，年化超6%应该很简单。
我最近玩玩大a，看到你的缠论，这个形态正是我想尝试做的，我今天吧代码down下来，学习一下

**Comment by Vespa314** [comment]
个人几期实验下来，发现美股会被A股的『可预测性』好很多。。

**Reply by benjaminlihao** [reply]
美股是讲规则的，大a的政策影响很大，人为干预权重太大
